doc/api/modules.md

```diff
@@ -233,6 +232,23 @@ This property is experimental and can change in the future. It should only be us
 by tools converting ES modules into CommonJS modules, following existing ecosystem
 conventions. Code authored directly in CommonJS should avoid depending on it.
 
+To create an ESM module that should provide an arbitrary value to CommonJS, the
+`__cjsUnwrapDefault: true` marker can be used instead:
```

Comment:
The documentation is still using the prefix. Also I think the documentation should be explicit about the fact that when it's defined, the named exports will get lost in the `require()` result, so if library authors want to maintain

```js
const { method } = require('unwrappedESM');
```

`method` must be put on the default export object.

Reply:
Thanks, added.